src/monsters.h

```diff
@@ -181,8 +196,8 @@ class Monsters
 
 		std::map<std::string, uint32_t> monsterNames;
 		std::map<MonsterType*, std::string> monsterScriptList;
-		std::map<uint32_t, MonsterType*> monsters;
-		LuaScriptInterface* scriptInterface;
+		std::map<uint32_t, MonsterType> monsters;
```

Comment:
I just thought you should put that in the future changelog for 1.2. Something tells me someone actually did use that even though it makes no sense...